content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -207,8 +207,8 @@ for i in ret.items:
 ```
 
 #### Other languages
-
-There are [client libraries](/docs/reference/using-api/client-libraries/) for accessing the API from other languages. See documentation for other libraries for how they authenticate.
+content/en/docs/reference/using-api/client-libraries.md
+There are [client libraries](/website/content/en/docs/reference/using-api/client-libraries/) for accessing the API from other languages. See documentation for other libraries for how they authenticate.
```

Comment:
I do not think this is the correct link... Original was the correct link https://kubernetes.io/docs/reference/using-api/client-libraries/

Reply:
weird
so these kind of links works on the website but not working through github pages
works fine through here : 
https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-api/